pkg/cmd/server/start/start_master.go

```diff
@@ -451,7 +454,20 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 
 	var standaloneAssetConfig *origin.AssetConfig
 	if oc.WebConsoleEnabled() {
-		config, err := origin.BuildAssetConfig(*oc.Options.AssetConfig)
+		var overrideConfig *overrideapi.ClusterResourceOverrideConfig = nil
+		if overridePluginConfigFile, err := pluginconfig.GetPluginConfigFile(oc.Options.KubernetesMasterConfig.AdmissionConfig.PluginConfig, "ClusterResourceOverride", ""); err != nil {
```

Comment:
Yours is the first needing this kind of a ref.  Let's go with a constant for the magic string to ensure we don't slip.